notebook/databricks/xgboost_notebook_mortgage-gpu_scala.scala

```diff
@@ -0,0 +1,128 @@
+// Databricks notebook source
```

Comment:
What's the difference b/w databricks notebook vs. our existing notebooks?

Reply:
databricks imports a .ipynb file (which is what those notebooks are) as a python notebook and its really a scala one.  I'll play with it a bit to see if I can change the type or force import

Reply:
yeah I don't see anyway to force it imported as scala.  We can try to ask them but Karthik said they were out yesterday.
I copied the existing notebook so the contents other then the path to the data is the same across them.